README.md

```diff
@@ -1 +1,51 @@
-# java-jaxrs
\ No newline at end of file
+# OpenTracing JAX-RS Instrumentation
+
+Warning: This library is still a work in progress!
+
+[![Travis](https://travis-ci.org/opentracing-contrib/java-jaxrs.svg?branch=master)](https://travis-ci.org/opentracing-contrib/java-jaxrs)
+
+OpenTracing instrumentation for JAX-RS standard. 
+It supports server and client request tracing.
+
+## Tracing Server Requests
+```
+// register this in javax.ws.rs.core.Application
+ServerTracingDynamicFeature.Builder
+    .traceAll(yourPreferredTracer)
+    .withOperationNameProvider(ServerOperationNameProvider.HTTP_METHOD_NAME_PROVIDER)
```

Comment:
Operation name can be modified by `SpanDecorator` so it can be confusing. I can think of this solutions:

1. allow `SpanDecorator` to modify only tags, logs, baggage(?) and keep `OperationNameProvider`.
2. get rid of `OperationNameProvider` and use only decorators which operate on span.

What do you think?
I think that second one is more flexible.